pages/workspaces/[workspaceId]/contributor-insights/new.tsx

```diff
@@ -147,6 +147,8 @@ export default function CreateContributorInsightPage({
               variant="primary"
               className="flex gap-2.5 items-center cursor-pointer w-min mt-2 sm:mt-0 self-end"
               disabled={loading}
+              loading={loading}
+              loadingText={"Creating Insight.."}
```

Comment:
refactor (blocking): No need to appen `..`. Also we lowercase the second and later words in buttons.

```suggestion
              loadingText={"Creating insight"}
```

Reply:
got it, thanks